test/TestHelper.js

```diff
@@ -10,6 +10,7 @@ const mockyeah = require('../index.js');
 const request = require('supertest');
 
 const configHttps = Object.assign({}, mockyeah.config, {
+  port: undefined,
```

Comment:
Assigning `undefined` can be ambiguous, thoughts on `null`?

Reply:
In a real use case, someone using https port would not define port, so undefined is appropriate here.

Besides, lately I never like to use null if I can avoid. The defaulting syntax of modern JavaScript makes undefined so much nicer, since it doesn't work with null. Even Crocked thinks there should only be either undefined or null and not both, though he liked null a while back because it's shorter...

Reply:
Thank you for justifying.